### PR TITLE
Update english.ts

### PR DESCRIPTION
**Pull request by XCOM-HUB**
I used Grammarly to correct grammatical errors and make the English easier to understand. I hope you like it.

### CODE REVIEW

**Review by kambala-decapitator — COMMENTED** [review]
everything else looks good

**Comment by IvanSavenko** [comment]
Thanks for spellcheck, but these changes should be integrated directly in source code, e.g. in cpp files that define these strings in the first place.

Will take a look a bit later.

File: launcher/translation/english.ts [hunk 1]
```diff
@@ -1194,8 +1194,8 @@ Heroes® of Might and Magic® III HD is currently not supported!</source>
     </message>
     <message>
         <location filename="../firstLaunch/firstlaunch_moc.ui" line="504"/>
-        <source>If you own Heroes III on gog.com you can download backup offline installer from gog.com, and VCMI will import Heroes III data using offline installer. 
-Offline installer consists of two parts, .exe and .bin. Make sure you download both of them.</source>
+        <source>If you own Heroes III on gog.com, you can download a backup offline installer from gog.com. VCMI will then import Heroes III data using the offline installer. 
+Offline installers consist of two parts, .exe, and .bin. You need to download both of them.</source>
```
[the remark this answers is not in the document]
Comment (XCOM-HUB) [reply launcher/translation/english.ts:1198]: I made line 1198 even better ➡️ Offline installer consists of two files: ".exe" and ".bin" - you must download both.

File: launcher/translation/english.ts [hunk 2]
```diff
@@ -1339,7 +1339,7 @@ Offline installer consists of two parts, .exe and .bin. Make sure you download b
     </message>
     <message>
         <location filename="../firstLaunch/firstlaunch_moc.cpp" line="383"/>
-        <source>You&apos;ve provided GOG Galaxy installer! This file doesn&apos;t contain the game. Please download the offline backup game installer!</source>
+        <source>You've provided a GOG Galaxy installer! This file doesn't contain the game. Please download the offline backup game installer!</source>
```
[the remark this answers is not in the document]
Comment (XCOM-HUB) [reply launcher/translation/english.ts:1342]: Ok! It will show correctly in the Launcher, I guess!?

**Comment by IvanSavenko** [comment]
Closing, integrated changes into #5000, will be merged as part of that PR, to avoid changing translation files multiple times (and resolving potential conflicts in them)

**Pull request closed without merging**